components/Days.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <div class="week-box">
+    <div class="heading">
+      <img src="~/assets/ankur.png" alt="userpic" />
+      <h1>Your goals</h1>
+    </div>
+    <h2 class="week-heading">This week</h2>
+    <div class="days">
+      <div class="day-box">
```

Comment:
Could have maintained an array for the days with their completion status, and mapped it.
It shouldn't have been this verbose.
.
Just a suggestion.

Reply:
done